plugins/aea-cli-ipfs/aea_cli_ipfs/core.py

```diff
@@ -39,10 +39,10 @@
 
 @click.group()
 @click.pass_context
-@click.option("--online", is_flag=True)
-def ipfs(click_context: click.Context, online: bool) -> None:
+@click.option("--offline", is_flag=True, default=False)
```

Comment:
from now we should use the online node as default.

Reply:
The one we host on the cluster?
If a package is not found on the remote, does it then search the local or as a dev does this need to be specified?

Reply:
If a package is not found on the remote we can always use --offline flag and retry

Reply:
It would be nice if there was automatic fallback to trying --offline after --online fails. that's how the core registry works too.

Reply:
But this could be done in a separate PR and an issue opened for it.